server/supabase/migrations/20240608204813_helper_party.sql

```diff
@@ -0,0 +1,76 @@
+create table
+helper_parties (
+uuid uuid default gen_random_uuid() primary key,
+display_name varchar(255) unique not null,
+created_at timestamp default current_timestamp not null
```

Comment:
the standard way is to define both `created_at`, `modified_at` and `modified_by`

Reply:
adding `modified_at`. I'll figure out `modified_by` (and I suppose `created_by`) later.